lib/_tls_wrap.js

```diff
@@ -1169,10 +1161,11 @@ function Server(options, listener) {
   if (typeof options === 'function') {
     listener = options;
     options = {};
-  } else if (options == null || typeof options === 'object') {
-    options = options || {};
   } else {
-    throw new ERR_INVALID_ARG_TYPE('options', 'Object', options);
+    if (options !== undefined) {
+      validateObject(options, 'options', { nullable: true });
+    }
+    options = options || {};
```

Comment:
```suggestion
    options ??= {};
    validateObject(options, 'options');
```

or even better:

```js
} else if (options == null) {
  options = {};
} else {
  validateObject(options, 'options');
}
```

Reply:
Great

```
} else if (options == null) {
  options = {};
} else {
  validateObject(options, 'options');
}
```

This makes the code much clearer and easier to understand